project/plugins.sbt

```diff
@@ -2,3 +2,4 @@ addSbtPlugin("se.marcuslonnberg" % "sbt-docker" % "1.9.0")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "1.1.0")
 addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.2")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.9.3")
+addDependencyTreePlugin
```

Comment:
Worth keeping in even after the investigation is complete?

Reply:
I think so, I only learned about `whatDependsOn` _today_ after years of scrolling through endless output from `dependencyTree`. Any reminder to myself or others to use the efficient thing is a win in my book.